### PR TITLE
Features/new html tags

### PR DESCRIPTION
**Pull request by Jan-Luca2507**

### CODE REVIEW

File: public/index2.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
+<html lang="de">
+
+<head>
+    <script>
```
Comment (codeministry) [inline public/index2.html]: Pls remove JS here, not necessary - also in `index3.html`

File: public/index.html [hunk 2]
```diff
@@ -113,13 +110,16 @@ <h2>das ist die unterueberschrift</h2>
...
         </div>
         <div>
             <a href="https://wiki.selfhtml.org/wiki/HTML/Elemente/a" target="_blank">Hier geht's zu selfhtml</a>
         </div>
+        <div>
+            <a href="index2.html">Hier geht es zu site two</a>
+        </div>
+        <div>
+            <a href="index3.html">Hier geht es zu site three</a>
+        </div>
```
Comment (codeministry) [inline public/index.html:123]: Missing `</div>`, please also check the other HTML files.

File: public/index2.html [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+       </header>
+    </div>
+    <hr>
+      <div>
+    <nav>
+           <h2>
+               das ist die ueberschrift
+           </h2>
+           <ul>
+            <li>https://wiki.selfhtml.org/wiki/HTML/Elemente/nav 1</a></li>
```
Comment (codeministry) [inline public/index2.html]: 1. It's correct, but such links should be implemented as links - also check the other HTML-templates
2. What does the `1` mean?

File: public/index2.html [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+        <article>
+            <h1>das ist "address"</h1>
+            <p>Lorem ipsum dolor sit amet, consetetur</p>
+            <address>
+                Autor: <a href="Lorem ipsum dolor sit amet, consetetur">Max Mustermann</a>
+            </address>
+        </article>
+        <footer>
+            <adress>
+                © 1998-2013 SELFHTML <a href="https://selfhtml.org/impressum.html">Impressum</a> Software: Classic Forum 3.4
```
Comment (codeministry) [inline public/index2.html]: Please replace the `©` with a corresponding HTML entity (https://wiki.selfhtml.org/wiki/Zeichenreferenz)

File: public/index3.html [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+            <form>
+                <label for="vname">Vorname:
+                    <input id="vname" name="vname">
+                </label>
+                <label for="zname">Zuname:
+                    <input id="zname" name="zname">
+                </label>
+
+                <label for="m&auml;nnl">m&auml;nnlich</label>
+                <input type="radio" id="m&auml;nnl" name="geschlecht" value="0">
```
Comment (codeministry) [inline public/index3.html]: - Use english words for IDs, names, classes etc.
- Don't use HTML entities in IDs, names, classes, etc.

File: public/index3.html [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+                <button type="submit">Eingaben absenden</button>
+            </form> 
+        </div>
+        <hr>
+        <hr>
+        <div>
+            <form action="select.html">
+                <label>K&uumlnstler(in):
+                    <select name="top5" size=4>
+                        <option>Michael Jackson</option>
```
Comment (codeministry) [inline public/index3.html]: Options should also have a `value` attribute

File: public/index.html [hunk 7]
```diff
@@ -82,10 +82,7 @@ <h2>das ist die unterueberschrift</h2>
             <p>
                 <code>das hier ist der code</code>
             </p>
-            <video src="Demofilm.mp4" controls>
-                <br>
-                <a href="https://www.youtube.com/watch?v=FYH9n37B7Yw"></a>
-            </video>
+            <video src="videos/demofilm.mp4" controls></video>
```
Comment (codeministry) [inline public/index.html]: Please use CSS to resize with of video to `300px`

File: public/index3.html [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+                        Zuname: <input name="zuname" id="zuname" placeholder="Mustermann">
+                    </label>
+
+                </fieldset>
+
+                <fieldset>
+                    <legend>Geschlecht</legend>
+
+                    <label for="m&aumlnnl">
+                        <input type="radio" name="geschlecht" id="m&aumlnnl" value="m&aumlnnl">
```
Comment (codeministry) [inline public/index3.html]: Use english words for attributes, labels, etc.

File: public/index2.html [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+                </tr>
+            </tbody>
+            <tfoot>
+                <tr>
+                    <th>wetter</th>
+                    <td>sonnig</td>
+                    <td>bewoelkt</td>
+                </tr>
+                <tr>
+                    <th> colspan="3">Alle Angaben sind mega cool</th>
```
Comment (codeministry) [inline public/index2.html]: Wrong HTML syntax, check screenshot:

Bildschirmfoto 2021-06-24 um 09 15 34

File: public/index2.html [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+            <thead>
+                <tr>
+                    <th></th>
+                    <th>hamburg</th>
+                    <th>berlin</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <th>montag</th>
```
Comment (codeministry) [inline public/index2.html]: Use `th` only in header

File: public/index2.html (hunk 3 above)
Comment (Jan-Luca2507) [reply public/index2.html]: die 1 war ein tippfehler

File: public/index.html (hunk 7 above)
Comment (Jan-Luca2507) [reply public/index.html]: Ich hätte die größe mit `height="size-300" width="size-300"` verändert aber das funktioniert nicht

**Pull request merged**